OctopusDSC/Tests/cOctopusServerSslCertificate.Tests.ps1

```diff
@@ -0,0 +1,127 @@
+#requires -Version 4.0
+[System.Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSAvoidUsingConvertToSecureStringWithPlainText', '')] # these are tests, not anything that needs to be secure
+param()
+
+$moduleName = Split-Path ($PSCommandPath -replace '\.Tests\.ps1$', '') -Leaf
+$modulePath = Split-Path $PSCommandPath -Parent
+$modulePath = Resolve-Path "$PSCommandPath/../../DSCResources/$moduleName/$moduleName.psm1"
+$module = $null
+
+try
+{
+    $prefix = [guid]::NewGuid().Guid -replace '-'
+    $module = Import-Module $modulePath -Prefix $prefix -PassThru -ErrorAction Stop
+
+    InModuleScope $module.Name {
+        $sampleConfigPath = Split-Path $PSCommandPath -Parent
+        $sampleConfigPath = Join-Path $sampleConfigPath "SampleConfigs"
+
+        Describe 'cOctopusServerSslCertificate' {
+            Context 'Get-TargetResource - when present and port not specified' {
```

Comment:
Should be another `Describe` block for `Get-TargetResource`, and you can then remove that from the name of the context blocks.

Reply:
I was following the pattern in cOctopusServerSpace.Tests.ps1, is that one also incorrect?

Reply:
It's not so much "incorrect", as "I'd say my suggested approach here is slightly better"